web/src/components/login-page/index.tsx

```diff
@@ -101,7 +101,7 @@ export const LoginPage: FC = memo(function LoginPage() {
               />
               {isPlayEnvironment && (
                 <div className={classes.note}>
-                  Try with <strong>play</strong> name if you want to join the
+                  Try project name <strong>play</strong> if you want to join the
```

Comment:
or just 😄 

```
Enter <strong>play</strong> if you want to join the playground environment
```

Reply:
`Enter` is a bit odd to me, so lets me keep it as is 😄

Reply:
But, maybe `Input <strong>play</strong> if you want to join the playground environment` is good. How do you think? @nghialv @knanao

Reply:
Looks good!

Reply:
addressed by https://github.com/pipe-cd/pipecd/pull/3424/commits/2e02521956bcdbe44baf69112c9fcc1d952969d0 🙌